src/ui/Crafting.svelte

```diff
@@ -37,10 +37,13 @@
 <style>
 	#Crafting {
 		position: absolute;
-		top: 0px;
-		width: 100%;
-		height: 100%;
-	  pointer-events: fill; /* Allows the buttons to be clickable */
+		top: 20px;
+		left: 20px;
+		pointer-events: auto; /* Allows the buttons to be clickable */
```

Comment:
@ACarr0ll I didn't know about `fill`, I found [this](https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events) and changed it to `auto` but wanted to make sure I did the right thing, in case you made it `fill` intentionally

Reply:
@AphLute Totally fine! I had it set to auto but switched to fill but misread the documentation. Fill is for SVG only which we are not working with at all. Good catch!